configure.ac

```diff
@@ -153,12 +153,18 @@ fi
 AC_DEFINE_UNQUOTED(CONF_PATH_RANDOMDEV, "$opt_randomdev", [Random device path.])
 
 dnl Check for cracklib
-AC_CHECK_HEADERS([crack.h],
-      AC_SEARCH_LIBS([FascistCheck], [crack], LIBCRACK="-lz -lcrack" LIBS="$LIBS -lz", LIBCRACK="", [-lz]))
-if test "x$LIBCRACK" = "x" ; then
-    AC_MSG_ERROR([No or unusable cracklib library])
+AC_ARG_WITH([cracklib], AS_HELP_STRING([--without-cracklib],[disable cracklib]), [], [opt_cracklib=yes])
```

Comment:
Please change this to AC_ARG_ENABLE and the description should be `disable cracklib dictionary check` or something like that.

Reply:
Thank you for the suggestion, I will change it.

Sorry for the mistake, I am not proficient with Autoconf and used what I already seem on this file.

Reply:
@t8m any problem the parameter should be `--disable-cracklib` ?

Reply:
Hmm, maybe `--disable-cracklib-check` so the enable should be for cracklib-check.